source/_cookbook/customize_polling_interval.markdown

```diff
@@ -7,8 +7,8 @@ sidebar: true
 comments: false
 sharing: true
 footer: true
-ha_category: Example configuration.yaml
----
+ha_category: Customize Defaults
+-------------------------------
```

Comment:
Why did you add these extra dashes? Are you sure that jekyll picks that up correctly?